### DISCUSSION TITLE
Unable to get Binance and OKEx futures data

### DISCUSSION THREAD

**Discussion by MarCorCon** [bmoscon/cryptofeed#557] [Q&A] [answered]
Hi all! I am simply amazed by this project! It is incredible how easy is to get all the data from all the exchanges! 

I am getting the BTCUSD trades from many exchanges, but I am not able to get the Binance Futures and OCEx.
For Binance, I try with:

`f.add_feed(BinanceFutures(subscription={TRADES: ['BTCUSDT']}, callbacks={TRADES: self.new_trade}))`

but I get: 
`cryptofeed.exceptions.UnsupportedSymbol: BTCUSDT is not supported on BINANCE_FUTURES`

I also don't understand how to subscribe to OKEx futures data for BTC. I only get spot data.

**Comment by toyan** [comment]
Try to use `'BTC-USDT'` instead of `'BTCUSDT'`

**Comment by bmoscon** [comment, marked as answer]
@MarCorCon - use the `symbols()` method on the OkEx exchange class. It will show you the supported symbols
